zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/InterpreterResult.java

```diff
@@ -46,7 +46,8 @@
     TABLE,
     IMG,
     SVG,
-    NULL
+    NULL,
+    NETWORK
```

Comment:
Hi,
i updated the code base in order to manage simple graphs like this:
```
%spark
print(s"""
%network {
    "nodes" : [
        {"id" : 1},
        {"id" : 2}
    ],
    "edges" : [{"source" : 2, "target" : 1, "id" : 1 }]
}
""")
```
![neo4-simple-graph](https://cloud.githubusercontent.com/assets/1833335/20022647/c5c383b4-a2db-11e6-81d2-eece1b0923bd.png)